src/MapViewController.swift

```diff
@@ -81,6 +206,7 @@ open class MapViewController: UIViewController, LocationManagerDelegate {
   open var currentAnnotations: [PeliasMapkitAnnotation : TGMapMarkerId] = Dictionary()
   open var attributionBtn = UIButton()
 
+  /// The camera type we want to use. Defaults to standard top-down (flat) camera.
```

Comment:
Not sure we can assume a default here, it is a defaulted from whatever is in the scene if unset.